Transmission.xcodeproj/project.pbxproj

```diff
@@ -1498,6 +1498,7 @@
 				1BB44E07B1B52E28291B4E30 /* file-piece-map.cc */,
 				1BB44E07B1B52E28291B4E31 /* file-piece-map.h */,
 				C1425B321EE9C5EA001DB85F /* tr-assert.cc */,
+				E7DA3781279AE0B700E261D8 /* tr-assert.mm */,
```

Comment:
CMake-based build will fail on Mac as the file wasn't added to corresponding CMakeLists.txt